src/main.c

```diff
@@ -88,6 +91,13 @@ int load_conf_file(char *file_path)
                 vector_append(&tests_to_run, strtol(current_test, NULL, 16));
             }
         }
+        if (strcmp("tests-exclude", current_key) == 0) {
+            char *current_test;
+            char *tests = strtok(NULL, NEWLINE_DELIMITER);
+            while ((current_test = strtok_r(tests, ",", &tests))) {
+                vector_append(&tests_exclude, strtol(current_test, NULL, 16));
```

Comment:
Would it be possible to alternatively specify tests to exclude _by name_ instead of (or in addition to) by number?

Reply:
That will be a feature request and will be able to support in a separate branch. Which I had reworked the global tests to remove user error mistakes. Plus no longer need to manually add hex value and name of api strings for each tests. So, like I said, your suggestion is a feature request.

Currently, it's following same way for tests input.

Reply:
Transfer into ticket #100